changed `plotrange`

knuppla opened the pull request:
Changed `plotrange` to be dependent on minimum concentration value. Left maximum `plotrange` value at 100 to maintain a wide view of the plot but can be easily changed later. Added untreated control group check and pseudo transformation found in concRespPlot2.

knuppla committed:
added pseudo value transformation

knuppla committed:
changed max to vary with experimental concentration values

Kelly-Carstens-EPA reviewed (commented):
It looks like the min/max experimental conc feature is working. I did notice that the curve was not fitting for concs below 1e-3. For example: 
![image](https://github.com/user-attachments/assets/2b13fb20-8adb-4445-bd94-9bcea38b889c)

knuppla committed:
Removed hard-coding of plotting grid points

knuppla commented:
I removed the hard-coding of the plotting grid points entirely. I replaced `conc_plot` with the provided experimental concentrations. I also compared the plot against concRespPlot2. 
concRespPlot
![smallConcsconcRespPlot](https://github.com/user-attachments/assets/640e2092-b97f-4a91-9c16-87540059df1f)

concRespPlot2
![smallConcsconcRespPlot2](https://github.com/user-attachments/assets/bfdba3b1-982f-4ea1-922e-a28823d85a8a)

knuppla committed:
changed seq generation in conc_plot

Kelly-Carstens-EPA reviewed (commented):
I also replicated the straight line with other data. It looks like I can still get a curve but it gets dropped in some cases. Maybe @sedavid01 has an idea.

![image](https://github.com/user-attachments/assets/e582f2fa-27c6-4aab-a806-b4c4bfc63516)
![image](https://github.com/user-attachments/assets/7b2c0bd0-5b67-4feb-bb91-6a847633641a)

knuppla committed:
changed grid point generation for curve fit plotting
